src/Statistics/Repositories/TopScoreRepository.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Statistics\Repositories;
+
+use VSV\GVQ_API\Statistics\TopScore;
+use VSV\GVQ_API\User\ValueObjects\Email;
+
+interface TopScoreRepository
+{
+    public function findByEmail(Email $email): ?TopScore;
```

Comment:
Onze conventie van naamgeving wijkt wat af van Doctrine en we gebruiken `getByEmail`

Reply:
OK, ik ben gewoon van `find` te gebruiken als er null kan terug komen en `get` wanneer er een _exception_ mogelijk is.